cli/testdataseeder.php

```diff
@@ -1,28 +1,28 @@
 <?php
 define('CLI_SCRIPT', 1);
```

Comment:
Add header things before define.

Reply:
Good point.
This has already been merged, so we should address this when @mlynn-lp provides his next PR on this script.